5-infrastructure/business_unit_1/non-production/main.tf

```diff
@@ -27,9 +27,9 @@ module "env" {
   boa_ops_project_id           = var.boa_ops_project_id
   boa_sec_project_id           = var.boa_sec_project_id
   boa_sql_project_id           = var.boa_sql_project_id
-  gke_cluster_1_cidr_block     = var.gke_cluster_1_cidr_block
-  gke_cluster_2_cidr_block     = var.gke_cluster_2_cidr_block
-  gke_mci_cluster_cidr_block   = var.gke_mci_cluster_cidr_block
+  gke_cluster_1_cidr_block     = "100.64.142.0/28"
```

Comment:
can we add a comment here referencing to the network section so the user knows how they are related